docs/pages/examples.mdx

```diff
@@ -0,0 +1,75 @@
+     - receiver address
+     - payload (arbitrary bytes)
+     - attributes (optional protocol-specific parameters)
+   - Returns an outbox ID to track the message
+   - Emits a `MessagePosted` event
+
+2. `IERC7786Receiver` - The destination chain interface that receives messages. Key components:
+   - `executeMessage`: Function called by the destination gateway to deliver the message
+   - Must verify the calling gateway is trusted
+   - Must return a specific selector (0x675b049b) to confirm successful execution
+
+Important concepts to understand:
+
+- Messages require proper setup of source and destination gateways that implement these interfaces
+- Chain identifiers follow the CAIP-2 standard (e.g. "eip155:421614" for Arbitrum Sepolia)
+- Some protocols (like Axelar) may require additional steps like:
+  - Registering chain equivalences between protocol-specific and CAIP-2 identifiers
+  - Registering remote adapter addresses
+  - Paying for destination chain gas fees and potentially relayer or protocol fees
+  - Message approval and execution on the destination chain
+
+The standard is designed to be protocol-agnostic while allowing protocol-specific features through the attributes parameter.
+
+### Setup
+
+1. The first step is to deploy the adapters on source and destination chains. Adapters can be receive-only, send-only or work both ways. In order to be able to send messages in 
+both directions, we deployed [`AxelarGatewayDuplex.sol`](https://github.com/OpenZeppelin/openzeppelin-community-contracts/blob/master/contracts/crosschain/axelar/AxelarGatewayDuplex.sol) in both chains.
+Note that when deploying the contract, one of the constructor parameters is the Axelar gateway proxy, which is the contract responsible for receiving (or sending) all messages in the respective chain. 
+Note that each chain has its own gateway proxy (at least one). This archicteture is used to centralize all messages in one place, where relayers can easily listen for events. 
+This is equivalent to LayerZero Endpoints and Hyperlane Mailboxes, for example. The contract addresses for the gateways on different chains for Axelar can be seen at its [docs](https://docs.axelar.dev/resources/contract-addresses/testnet/).
+    
+    - Contract deployed on source chain: [0xBb6A11d2FdEF18522cC9d14DdAdb676c8c126AEE](https://sepolia.basescan.org/address/0xBb6A11d2FdEF18522cC9d14DdAdb676c8c126AEE)
+    - Contract deployed on destination chain: [0x050193203845D4145Dea5200C7eA49EF28b5f870](https://sepolia.arbiscan.io/address/0x050193203845d4145dea5200c7ea49ef28b5f870)
+
+2. With the contracts deployed (source and destination chains), we need to register the chain equivalence between axelar ids and the ERC7786 standard. Note that when sending a message, the destination chain [parameter](https://github.com/OpenZeppelin/openzeppelin-community-contracts/blob/cc554fc5cfee7498dc0177beb01118e4499826f6/contracts/crosschain/axelar/AxelarGatewaySource.sol#L29) 
```

Comment:
```suggestion
2. With the contracts deployed (source and destination chains), we need to register the chain equivalence between axelar ids and the ERC7786 standard. This is because when sending a message, the destination chain [parameter](https://github.com/OpenZeppelin/openzeppelin-community-contracts/blob/cc554fc5cfee7498dc0177beb01118e4499826f6/contracts/crosschain/axelar/AxelarGatewaySource.sol#L29) 
```